server/http4s-server/src/main/scala/tapir/server/http4s/EndpointToHttp4sServer.scala

```diff
@@ -96,7 +110,20 @@ object EndpointToHttp4sServer {
 
       val response: ContextState[F] = matchInputs[F](inputs)
 
-      val value: F[Either[Error, (Context[F], MatchResult[F])]] = context.map(response.run)
+      val methodMatches: Either[String, Error] =
```

Comment:
shouldn't this be `Either[Error, String]` ?

Reply:
Yes you're right, `Error` just happens to be `String` by "lucky" coincidence xD I'll fix this